src/cljdoc/render/error.clj

```diff
@@ -7,7 +7,8 @@
 (defn not-found-404 []
   (->> [:div.pt4
         [:div.mt5-ns.mt6-l.mw7.center.pa4.pa0-l
-         [:p.f2-ns.f3.mv3.w-90-l.lh-copy "We couldn't find anything here..."]
+         [:h1.f2.f1-ns.helvetica.b ":("]
```

Comment:
Why the `.helvetica` here? cljdoc usually uses `.sans-serif` which defaults to the OS' default sans serif font.

Reply:
I don't have specific reason. Just wanted to play around with different fonts to make it look a bit nicer. Let me put it back.

Reply:
Ok cool, was just wondering 🙂